pipelines/windows-efi-installer/README.md

```diff
@@ -0,0 +1,167 @@
+# Windows EFI Installer Pipeline
+
+This pipeline installs Windows 11/2k22 into a new DataVolume. This DataVolume is suitable to be used as a default boot source
+or golden image for Windows 11/2k22 VMs.
+
+This example pipeline is suitable only for Windows 11/2k22 (or other Windows versions which require EFI - not tested!). When using 
+this example pipeline always adjust pipeline parameters for Windows version you are currently using (e.g. different template name, 
+different autoattend config map, different base image name, etc.). Each Windows version requires change in autounattendConfigMapName 
+parameter (e.g. using `windows2k22-autounattend` config map will not work with Windows 11 and vice versa - e.g. due to different storage 
+drivers path).
+
+The pipeline implements this by modifying the windows ISO - extracts ISO files from ISO, replaces prompt bootloader with no-prompt bootloader. 
+This helps with automated installation of Windows in EFI boot mode. By default Windows in EFI boot mode uses a prompt bootloader, which will not 
+continue with the boot process until a key is pressed. By replacing it with the non-prompt bootloader no key press is required to boot into the 
+Windows installer. Then task packs updated packages to new ISO, converts it with qemu-img and replaces original ISO file in PVC.
+
+After the ISO is modified it creates a new VM which boots from the modified Windows installation image (ISO file). The installation of Windows is 
+automatically executed and controlled by a Windows answer file. Then the pipeline will wait for the installation to complete and will delete the 
+created VM while keeping the resulting DataVolume with the installed operating system. The pipeline can be customized to support different 
+installation requirements.
+
+## Prerequisites
+
+- KubeVirt `v1.0.0`
+- Tekton Pipelines `v0.44.0`
+- Apply ConfigMaps with Windows sysprep (or select one which you will need) - examples can be found here: https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/pipelines/windows-efi-installer/configmaps
+
+
+### Obtain Windows ISO 11 download URL
+
+1. Go to https://www.microsoft.com/en-us/software-download/windows11.
```

Comment:
Add a `getisourl.py` for it?

Reply:
It is already in the readme at the bottom